src/phpDocumentor/Descriptor/Builder/Reflector/Tags/SeeAssembler.php

```diff
@@ -38,23 +38,22 @@ public function create($data)
         $descriptor->setDescription($data->getDescription());
 
         $reference = $data->getReference();
+        $context = $data->getDocBlock() ? $data->getDocBlock()->getContext() : null;
 
         if (substr($reference, 0, 7) !== 'http://'
             && substr($reference, 0, 8) !== 'https://'
+            && substr($reference, 0, 6) !== 'ftp://'
             && $reference !== 'self'
             && $reference !== '$this'
+            && $reference[0] !== '\\'
         ) {
             // TODO: move this to the ReflectionDocBlock component
```

Comment:
Think this todo is not needed anymore since the code below has been changed? Or it should be moved to another line?